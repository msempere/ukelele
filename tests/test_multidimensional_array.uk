def main() void{
    int a[5][6];
    bool b[5];
    float c[1][2][3][4];
    a[1][2] = 3;
    println(a[1][2]);
    println(a[1][1]);
    b[1] = True;
    println(b[1]);
    println(b[2]);
    c[7] = 0.2;
    println(c[0]);
    println(c[7]);
}
