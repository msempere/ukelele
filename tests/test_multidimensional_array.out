3
0
True
False
0
0.2
